lib/fizzy/module.hpp

```diff
@@ -9,7 +9,7 @@
 
 namespace fizzy
 {
-inline const FuncType& function_type(const Module& module, FuncIdx idx)
+inline const FuncType& get_function_type(const Module& module, FuncIdx idx)
```

Comment:
```suggestion
inline const FuncType& get_function_type(const Module& module, FuncIdx idx) noexcept
```

Reply:
You can also make it a `Module`'s method now, but I'm not pushing this. It's rather blurry line what should be a method and what a free function.

Reply:
I'll make it a method, should it still be called `get_function_type` ?

Reply:
Yes.